perfkitbenchmarker/sample.py

```diff
@@ -57,12 +58,57 @@ class Sample(collections.namedtuple('Sample', _SAMPLE_FIELDS)):
     value: float. Result for 'metric'.
     unit: string. Units for 'value'.
     metadata: dict. Additional metadata to include with the sample.
+    timestamp: float. Unix timestamp.
   """
 
-  def __new__(cls, metric, value, unit, metadata=None, **kwargs):
+  def __new__(cls, metric, value, unit, metadata=None, timestamp=None,
+              **kwargs):
+    if timestamp is None:
+      timestamp = time.time()
+
     return super(Sample, cls).__new__(cls, metric, value, unit,
-                                      metadata=metadata or {}, **kwargs)
+                                      metadata=metadata or {},
+                                      timestamp=timestamp,
+                                      **kwargs)
 
   def asdict(self):
     """Converts the Sample to a dictionary."""
     return self._asdict()
+
+
+class SamplesTestMixin(object):
```

Comment:
@cmccoy I just uploaded a new version in which I create a file perfkitbenchmarker/test_util.py and put SamplesTestMixin in that file. I thought about it, and I don't see a reason to make a folder called "test_util" when there would only be one file in it. This way is also parallel to log_util.py and timing_util.py.